ssh/rootfs/etc/cont-init.d/ssh.sh

```diff
@@ -106,7 +106,7 @@ if [[ "${username}" != "root" ]]; then
         || bashio::exit.nok 'Failed adding user to wheel group'
 
     # Ensure new user switches to root after login
-    echo 'exec sudo -i' > "/home/${username}/.profile" \
+    echo 'exec sudo -i --preserve-env=SSH_AUTH_SOCK' > "/home/${username}/.profile" \
```

Comment:
Let's not solve this here, but in the sudoers configuration instead.

Reply:
Sure, done, updated commit pushed